pkgs/by-name/si/simpleBluez/package.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, dbus
+, fmt_9
+, lib
+}:
+let
+  pname = "simpleBluez";
+  version = "0.7.3";
+  src = fetchFromGitHub {
+    owner = "OpenBluetoothToolbox";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-CPBdPnBeQ0c3VjSX0Op6nCHF3w0MdXGULbk1aavr+LM=";
+  };
+in
+stdenv.mkDerivation {
+  inherit pname version src;
+
+  outputs = [
+    "out"
+    "dev"
+  ];
+
+  sourceRoot = "${src.name}/simplebluez";
+
+  cmakeFlags = [
+    "-DLIBFMT_LOCAL_PATH=${fmt_9.src}"
```

Comment:
Set `-DLIBFMT_VENDORIZE=OFF` and add `fmt` to `buildInputs` instead of this

Reply:
It says
```

Reply:
Moreover, do you believe using `fmt` instead of `fmt_9` is better? When enabling the vendorized version it uses [version 9](https://github.com/OpenBluetoothToolbox/SimpleBLE/blob/2eabe140b73bedc2a379d14eac14333693da377a/cmake/find/Findfmt.cmake#L13)